core/test/XiosFile_test.cpp

```diff
@@ -48,6 +51,10 @@ MPI_TEST_CASE("TestXiosFile", 2)
     std::unique_ptr<std::istream> pcstream(new std::stringstream(config.str()));
     Configurator::addStream(std::move(pcstream));
 
+    // Create a Model and configure it so that options are parsed
+    Model model(MPI_COMM_WORLD);
+    model.configureTime(); // TODO: Use Model.configure to parse restart files this way, too?
```

Comment:
To address this TODO (and the similar ones in the other tests), we'll need to align the config option used for restart files with and without XIOS. It would make sense to me to do so, rather than having separate XIOS-specific options. We want to make the user experience as similar to the non-XIOS case as possible.